src/lib/up-storage.js

```diff
@@ -88,6 +89,7 @@ class ProxyStorage implements IProxy {
     this.timeout = parseInterval(setConfig(this.config, 'timeout', '30s'));
     this.max_fails = Number(setConfig(this.config, 'max_fails', 2 ));
     this.fail_timeout = parseInterval(setConfig(this.config, 'fail_timeout', '5m' ));
+    this.strict_ssl = Boolean(setConfig(this.config, 'strict_ssl', true));
```

Comment:
It not should be false here?

Reply:
@juanpicado I believe strict_ssl should be true by default. Since this option was not available previously, only strict SSL was in effect. Therefore, using true as default ensures the same behavior the users are accustomed to.